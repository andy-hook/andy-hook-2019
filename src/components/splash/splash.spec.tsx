import React from "react"
import renderer from "react-test-renderer"

import Splash from "./splash"

const mockSocialData = [
  {
    node: {
      label: "twitter",
      url: "http://www.google.com",
    },
  },
  {
    node: {
      label: "linkedin",
      url: "http://www.google.com",
    },
  },
]

describe("<Splash />", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Splash socialIconData={mockSocialData} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
