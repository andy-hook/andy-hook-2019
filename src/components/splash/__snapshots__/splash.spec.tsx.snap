// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Splash /> renders correctly 1`] = `
<div
  className="sc-iwsKbI bcjQpo"
>
  <div
    className="sc-EHOje wOSEH"
  >
    <h1
      className="sc-bZQynM bAKAeH"
    >
      <test-file-stub
        className="sc-gzVnrw kzdxRF"
      />
      <span
        className="sc-htoDjs cIriUN"
      >
        Andy Hook
      </span>
      <span
        className="sc-dnqmqq iGobyL"
      >
        Interface Developer
      </span>
    </h1>
  </div>
  <div
    className="sc-gZMcBi gQGlAh"
  >
    <h2
      className="sc-gqjmRU ixFOao"
    >
      Iâ€™m busy working on something new
    </h2>
    <a
      className="sc-VigVT cCdqzR"
      href="http://www.google.com"
      target="_blank"
    >
      <span
        className="sc-jTzLTM efIpCi"
      >
        View some of my work
      </span>
    </a>
  </div>
  <div
    className="sc-fjdhpX kuTWWE sc-htpNat gXzayw"
  >
    <a
      className="sc-bxivhb hzcpoQ"
      href="http://www.google.com"
      target="_blank"
    >
      <div
        className="sc-ifAKCX XzGje sc-bdVaJa dqBkPn"
      >
        <svg
          className="icon sc-bwzfXH kdIDyY"
        >
          <use
            xlinkHref="/icon-sprite.svg#twitter"
          />
        </svg>
      </div>
    </a>
    <a
      className="sc-bxivhb hzcpoQ"
      href="http://www.google.com"
      target="_blank"
    >
      <div
        className="sc-ifAKCX XzGje sc-bdVaJa dqBkPn"
      >
        <svg
          className="icon sc-bwzfXH kdIDyY"
        >
          <use
            xlinkHref="/icon-sprite.svg#linkedin"
          />
        </svg>
      </div>
    </a>
  </div>
  <div
    className="sc-jzJRlG Swnsk"
  >
    <div
      className="sc-cSHVUG hpBCYf"
    />
    <test-file-stub
      className="sc-kAzzGY gfrgQr"
    />
  </div>
</div>
`;
